doc/classes/SceneTree.xml

```diff
@@ -217,7 +217,7 @@
 			<argument index="2" name="minsize" type="Vector2" />
 			<argument index="3" name="shrink" type="float" default="1" />
 			<description>
-				Configures screen stretching to the given [enum StretchMode], [enum StretchAspect], minimum size and [code]shrink[/code] ratio.
+				Configures screen stretching to the given [enum StretchMode], [enum StretchAspect], minimum size and [code]shrink[/code] (which can also be used to scale the content up).
```

Comment:
Would also be good to add a comment in the project setting docs that this can also be used for upscaling.

Reply:
Just to clarify, you are talking about godotengine/godot-docs/blob/stable//classes/class_projectsettings.rst, right? I am somewhat confused since so far I always assumed that this part of the docs was auto-generated, yet when I checked it (and the underlying ProjectSettings.xml file) just now there was no mention of any of the stretch settings. Am I missing something?

Reply:
I'm talking about `doc/classes/ProjectSettings.xml` where all project settings are _normally_ documented.

But here indeed the stretch settings are missing: https://github.com/godotengine/godot/blob/3.x/doc/classes/ProjectSettings.xml#L488-L490

That must mean that they are only defined under specific conditions which are not fulfilled when generating documentation with `doctool`. The `GLOBAL_DEF` calls should be moved somewhere else so that they appear in the docs. But that's not blocking for this PR then as it's only tangentially related.